src/omni-slider.js

```diff
@@ -1,532 +1,540 @@
-/* Constructor function
- * elementContainer - this acts as a wrapper for the slider, all of the sliders DOM elements will be transcluded inside the <element> provided
- * options - contains the options for the slider
- */
-function Slider(elementContainer, options) {
+(function() {
```

Comment:
open to not having a `use strict` or an `IIFE` at all. Babel is doing this for it anyway. @chrisrng?